seedtool/userinterface.ino

```diff
@@ -426,6 +513,9 @@ void seedy_menu() {
         yy += H_FSB9 + 2*YM_FSB9;
         g_display->setCursor(xx, yy);
         g_display->println("C - Wipe Seed");
+        yy += H_FSB9 + 2*YM_FSB9;
+        g_display->setCursor(xx, yy);
+        g_display->println("D - Xpub Keys");
```

Comment:
Should this be `D - Display XPUB` instead?

"Keys" is a little ambiguous, private? public? extended?  The other menu items all start with a verb.

Reply:
Fixed. I currently didn't include the "wipe" option, i will put it back once i figure out things with "display seed" option